django/core/handlers/base.py

```diff
@@ -246,6 +247,11 @@ def get_response(self, request):
 
         response._closable_objects.append(request)
 
+        # If the exception handler is a CBV, then force template to render.
+        if rendered is False:
+            if hasattr(response, 'render') and callable(response.render):
```

Comment:
I'd probably combine that with the outer `if`:

`if not rendered and hasattr(...) and callable(...):`